helper-tools/prepare-jamf-policy.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#-
+#- Usage
+#-   $ ./prepare-jamf-policy.sh "/path/to/Install macOS.app"
+#-   Wait seconds, getting checksumm...
+#-
+#-   =====================================================
+#-   Upgrade macOS Parameters:
+#-   Parameter 4: /Applications/Install macOS.app
+#-   Parameter 5: 10.14
+#-   Parameter 6: (Your download trigger policy)
+#-   Parameter 7: 20acadc8d66bb3d882bc0fedf1358c64
+#-   =====================================================
+#-
+#-   Would you need a package archive of Install macOS Mojave.app? [y/n]: y
+#-   Ok, building package archive file of Install macOS Mojave.app. Wait few minutes.
+#-   Done. ~/Desktop/Install macOS Mojave.app.10.14.pkg
+#-   $
+#-
+#-   You can get full-size installer with installinstallmacos.py.
+#-   https://github.com/munki/macadmin-scripts/blob/master/installinstallmacos.py
+#-
+
+if [ $# -eq 0 ]; then
+    # Show help
+    /usr/bin/grep ^#- "$0" | cut -c 4-
+    exit 0
+fi
+
+OSInstaller="$1"
+if [ ! -d "$OSInstaller" ]; then
+    echo "Not found $OSInstaller !"
+    exit 1
+fi
+
+if [ ! -d  "${OSInstaller}/Contents/SharedSupport" ]; then
+    echo "This installer looks like kind of 'stub' installer."
+    /usr/bin/du -sh "${OSInstaller}"
+    echo "Use full size intaller."
+    echo "You can get full-size installer with installinstallmacos.py"
+    echo "https://github.com/munki/macadmin-scripts/blob/master/installinstallmacos.py"
+    exit 1
+fi
+
+if [ ! -f  "${OSInstaller}/Contents/SharedSupport/InstallInfo.plist" ]; then
+    echo "Not found ${OSInstaller}/Contents/SharedSupport/InstallInfo.plist"
+    echo "Unknown installer type. Apple may change something."
+    exit 1
+fi
+
+if [ ! -f  "${OSInstaller}/Contents/SharedSupport/InstallESD.dmg" ]; then
+    echo "Not found ${OSInstaller}/Contents/SharedSupport/InstallESD.dmg"
+    echo "Thi is not expected installer type."
+    exit 1
+fi
+
+osversion=$(/usr/libexec/PlistBuddy -c "print 'System Image Info:version'" "${OSInstaller}/Contents/SharedSupport/InstallInfo.plist")
+echo "Wait seconds, getting checksumm..."
+checksum=$(/sbin/md5 -r "${OSInstaller}/Contents/SharedSupport/InstallESD.dmg" | /usr/bin/awk '{print $1}')
+
+cat <<_RESULT
+
+=====================================================
+Parameters for JamfPro policy:
+Parameter 4: /Applications/$( basename "$OSInstaller" )
```

Comment:
In some cases, the user might want to put it somewhere else, such as `/tmp` instead of `/Applications`. 
Therefore, it may not be appropriate to fix it to `/Applications`.

Reply:
Humm, I think a sample can not cover all cases. 
And to  fix to '/Applications' as a sample won't makes users strange in most cases. 
How about `(/Path/to/)` instead of `/Applications/` ?

Reply:
If we choice a path as `/Path/to/`, there is a contradiction in

> --root "${workdir}/root" 

 of https://github.com/kc9wwh/macOSUpgrade/pull/109/files#diff-6defd63a4f7c544c02a72f9ac5a11448R89.

Related to the https://github.com/kc9wwh/macOSUpgrade/pull/109/files#r283192864 discussion, we might be able to select following 2 ideas:

1. Adopt path as `/path/to`
2. Use this script consistently to create pkg without giving the user a choice

at this point.

Reply:
On my idea, take 2 "Use this script consistently to create pkg without giving the user a choice".
I'd like to make fix a place where the installer will be in /Applications.

Because,
-  a sample can not cover all cases. 
- a sample should give one major way.
- It is natural the installer is in /Applications, isn't it?
- In building package, it points to /Applicaitons. 
https://github.com/kc9wwh/macOSUpgrade/pull/109/files#diff-6defd63a4f7c544c02a72f9ac5a11448R82

Reply:
Sounds good. Your proposal is great. Please go ahead and proceed with it.